src/Frequency.cpp

```diff
@@ -107,7 +107,11 @@ boolean Frequency::handleSysex(byte command, byte argc, byte* argv)
 			  }
 			  if (internalMode >= 0)
 			  {
+#if defined (TARGET_RP2040) || defined (TARGET_RASPBERRY_PI_PICO)
+				  attachInterrupt(digitalPinToInterrupt(pin), FrequencyIsr, (PinStatus) internalMode);
```

Comment:
I think it would be better if this was also handled in boards.h, i.e. with a wrapper function. ifdefs in code files should be avoided.

Reply:
Fixed in the latest commit.